src/johari-submit/JohariSubmit.js

```diff
@@ -11,18 +11,18 @@ class JohariSubmit extends Component {
   }
 
   submit() {
-    fetch(`https://johariwindowapi.herokuapp.com/api/v1/johari/${this.props.evaluateeID}/descriptions`, {
+    fetch(`http://johariwindowapi.herokuapp.com/api/v1/users/${this.props.evaluateeID}/descriptions`, {
       method: 'POST',
       headers: {
         'Accept': 'application/json',
         'Content-Type': 'application/json',
       },
       body: JSON.stringify({
         johari: this.props.adjectives,
-        describer_id: '1',
+        describer_id: '150',
       })
```

Comment:
@kheppenstall Oh, this is hard-coded, I will make a chore to fix that after we add Auth0.